randovania/games/fusion/logic_database/Sector 6 NOC.txt

```diff
@@ -360,14 +365,28 @@ Extra - room_id: [8]
   * L0 Hatch to Clogged Cavern/Door to Nocturnal Playground
   * Extra - door_idx: (16,)
   > Door to Nocturnal Shaft
-      Trivial
+      Any of the following:
```

Comment:
I'm unsure of generator implications with this. Since it now gives the generator *another* point of no return here.
We'll keep it in for now, if issues arise again we'll adjust this.